documentation/src/docs/table-api.md

```diff
@@ -445,4 +488,38 @@ deployment:
   # (optional) port from cluster to gateway
   gateway-port: 0
 
-```
\ No newline at end of file
+```

```

Comment:
Add a section first to tell users that we can use SQL DDL to create table and bind with Pravega streams. Then comes the sample DDL and put a c,omment that the stream should be created in advance to have it working.